integration/service/test/test_cron_use_case.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Either discard or use in a checked test.

If adding to a test, please automate with a shorter report period